roles/cloud-bootstrap/tasks/accounts.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Ensure shared projects
+  os_project:
+    cloud: "{{ cloud }}"
+    name: "{{ item }}"
+  with_items:
+    - "{{ prod_project }}"
+    - "{{ nodepool_project }}"
+
+- name: Ensure user projects
+  os_project:
+    cloud: "{{ cloud }}"
+    name: "{{ item.username }}"
+  with_items: "{{ users }}"
+
+- name: Ensure groups
+  os_group:
+    cloud: "{{ cloud }}"
+    name: "{{ item }}"
+  with_items:
+    - "{{ prod_group }}"
+    - "{{ prod_admin_group }}"
+
+- name: Ensure users
+  os_user:
+    cloud: "{{ cloud }}"
+    name: "{{ item.username }}"
+    email: "{{ item.email }}"
+    default_project: "{{ item.username }}"
```

Comment:
This needs to set a password, defaulting to a simple one, like in the previous role. It also needs to set the task to not update the password on subsequent runs.

```
     password: "{{ default_password }}"
     update_password: on_create
```